api/src/main/java/org/pmiops/workbench/api/OfflineUserController.java

```diff
@@ -246,6 +250,10 @@ private int logCompletionChange(
     return logChange(user, oldTime, newTime, accessModuleLogText.get(module) + " completion");
   }
 
+  private int logAccessTierChange(DbUser user, List<String> oldTiers, List<String> newTiers) {
```

Comment:
tested locally with a local API+UI and enabling/disabling Google 2FA for my local user.